tfhe/src/core_crypto/algorithms/polynomial_algorithms.rs

```diff
@@ -712,7 +778,7 @@ mod test {
             let mut rng = rand::thread_rng();
 
             // random settings settings
-            let polynomial_log = (rng.gen::<usize>() % 7) + 6;
+            let polynomial_log = (rng.gen::<usize>() % 7) + 4;
```

Comment:
so the range changes from 6..12 to 4..10 ?

can we maybe expand it or keep it the same ? as we now have params with polynomial_log up to 16 IIRC ?